zebra-consensus/src/chain.rs

```diff
@@ -111,28 +111,25 @@ where
 
     fn call(&mut self, block: Arc<Block>) -> Self::Future {
         // TODO(jlusby): Error = Report, handle errors from state_service.
-        let span = tracing::debug_span!(
-            "block_verify",
-            height = ?block.coinbase_height(),
-            hash = ?block.hash()
-        );
-        let block_height = block.coinbase_height();
+        let height = block.coinbase_height();
+        let hash = block.hash();
+        let span = tracing::debug_span!("block_verify", ?height, ?hash,);
```

Comment:
I think this needs to be entered unless I'm missing where it's used later.
```suggestion
        let span = tracing::debug_span!("block_verify", ?height, ?hash,);
        let _guard = span.enter();
```

Reply:
I think you're right, we must have missed that in a much earlier review.

Reply:
Actually, no, the span is used to instrument the future:
https://github.com/ZcashFoundation/zebra/pull/989/files#diff-9ce8cedb3963941e6cffe1a307d9dafcL183

Reply:
aah yea there it is, just outside of the diff, lol